cumulus/ansible/tasks/providers/ec2.py

```diff
@@ -38,7 +38,11 @@ class EC2Provider(CloudProvider):
     }
 
     def __init__(self, profile):
-        super(EC2Provider, self).__init__(profile)
+        # super(EC2Provider, self).__init__(profile)
+        self.girder_profile_id = profile.get('_id', None)
+
+        for key, value in profile.items():
+            setattr(self, key, value)
 
```

Comment:
Code from the parent class was moved into the EC2Provider - calling init like this was causing all kinds of issues with mocking.